Formula/red.rb

```diff
@@ -0,0 +1,16 @@
+class Red < Formula
+  desc "Next-gen programming language, strongly inspired by REBOL"
+  homepage "http://www.red-lang.org"
+  url "http://static.red-lang.org/dl/mac/red-061"
```

Comment:
That's fine, we just want to build it from source. If that means we have to use a binary cross-compiler: so be it.